requirements-common.txt

```
@@ -6,7 +6,7 @@ requests >= 2.26.0
 tqdm
 blake3
 py-cpuinfo
-transformers >= 4.48.2  # Required for Bamba model and Transformers backend.
+transformers >= 4.49.0  # Required for Bamba and Zamba2 models and Transformers backend.
```

Typically we've been labeling this with the most recent model rather than appending to this line:
```suggestion
transformers >= 4.49.0  # Required for Zamba2 models and Transformers backend.
```